src/app/components/elements/panels/SelectedEventDetails.tsx

```diff
@@ -70,6 +70,10 @@ export const SelectedEventDetails = ({ eventId }: { eventId: string }) => {
                 </Badge>
               )}
             </p>
+            <p className="mb-0">
+              <strong>Hub: </strong>
+              {selectedEvent.hub ? hubNames[selectedEvent.hub] : "N/A"}
```

Comment:
is selectedEvent null safe?

Reply:
If hub comes over from the API, it's one of an enum. It's an optional parameter however, so it may be undefined, in which case it should show "N/A" here.